src/app/dialog-service/modal/modal/modal.component.html

```diff
@@ -1,12 +1,74 @@
-<div class="modal-header">
+<ng-template
+  [ngIf]="this.configData.type !== 'aboutModel'"
```

Comment:
instead if else here can we match conidtion inside the model  <div class="modal-body">
    <p>{{ configData?.label }}</p>
  </div>
  <div class="modal-footer">
    <button
      type="button"
      class="btn btn-outline-dark"
      (click)="activeModal.close(false)"
    >
      {{ configData?.no }}
    </button>
    <button
      type="button"
      class="btn btn-outline-dark"
      (click)="activeModal.close(true)"
    > here ? and show data according to conditions

Reply:
yes, i changed it.